### PR TITLE
New create keypair reference page

### PR DESCRIPTION
**Pull request by marians**
This adds a reference page for the `gsctl create keypair` command.

Pinging The Rossster for native speaker power 🇬🇧 !

### CODE REVIEW

File: content/reference/gsctl/create-keypair.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+date = "2017-06-19"
+type = "page"
+weight = 32
++++
+
+# `gsctl create keypair`
+
+The `gsctl create keypair` command creates a key pair and stores the related files in your certificate folder.
+
+If you intend to use the created key pair with the `kubectl` program, we recommend you to take a look at the [`create kubeconfig`](../create-kubeconfig/) command. It creates a new key pair and adds the according user, server, and context entries to your `kubectl` configuration.
```
Comment (puja108) [inline content/reference/gsctl/create-keypair.md]: <3

Comment (rossf7) [inline content/reference/gsctl/create-keypair.md]: This is good but "we recommend you take" reads more naturally for me.

**Comment by rossf7** [comment]
Late to the party but only found one minor 🇬🇧  issue :)

**Comment by marians** [comment]
Thanks! Fix is in https://github.com/giantswarm/docs-content/pull/244

**Pull request merged**